src/reducers/index.js

```diff
@@ -12,9 +12,15 @@ const initialState = {
 export default (state = initialState, action) => {
```

Comment:
You should be setting initialState in your reducer

const initialState = {
  count: 0
}